### ISSUE TITLE
Polish search for docs into Cloud.gov nav

### ISSUE THREAD

**Issue by suprenant** [cloud-gov/cg-site#451]
Per @thisisdano (originally in https://github.com/18F/cg-docs/issues/419):

A couple comments on this — not blocking release, but we'll want to have them eventually:
- highlighted search icon is appearing low-rez
- close box appearing low-rez
- close box should appear exactly where search icon appears (i.e. it shouldn't jump down)

**Comment by brittag** [comment]
To consider (moved from Slack discussion for decision by cg-style/visual-design folks):
- In search results, should the "visited" link color stay red, or should it move to purple to match the rest of the docs site?

**Comment by thisisdano** [comment]
I'll just rephrase this to say 'match docs visited link style in search results if possible'.

**Comment by brittag** [comment]
@thisisdano Yes! It looks like I can change this configuration in https://search.usa.gov/sites/6217/font_and_colors/edit - but I don't want to change colors without 👍  from cloud.gov visual design. :)

**Comment by thisisdano** [comment]
@brittag you get my official thumbs up

**Comment by brittag** [comment]
@thisisdano YAY thank you! Done!
